lib/theme_data.dart

```diff
@@ -109,4 +112,17 @@ class BreezColors {
     2: const Color.fromRGBO(0, 137, 252, 1.0),
     3: const Color.fromRGBO(0, 120, 253, 1.0),
   };
+}
+
+class VendorColors {
```

Comment:
I like the idea of expressing the vendor style in the theme but it is better no to use static variables.
One option is to define "VendorTheme" class with its properties (iconBgColor, iconFgColor ...) and instantiate one for each vendor. We can also add a map of vendor name to VendorTheme on the main theme object.